src/components/header/ControlOption.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+function ControlOption(props){
+
+    const { item } = props;
+    const { className, img, label, superScript} = item;
+
+    return (
+        <div className={`inline control_options ${className}`}>
+            <img src={img} alt={label} />
+            {label}
+            <span>{superScript}</span>
```

Comment:
Use conditionals because for every piece of data, you don't have a superScript and an empty span will be rendered for rest of the items